### DISCUSSION TITLE
Is there a way to have a package declared as private in monorepos

### DISCUSSION THREAD

**Discussion by dbismut** [yarnpkg/berry#2601] [Q&A] [answered]
Hi!

Forgive me if this is nonsensical, but it seems like yarn v2 removes `private: true` from `package.json files` in packages belonging to a monorepo. Is this the intended behavior or am I missing something?

Thanks a lot!

**Comment by n1ru4l** [comment, marked as answer]
When you set it to `"private": "true"` it is indeed stripped out, which is super weird behavior.  `"private": true` seems to work as expected and is not stripped out.
